content/general/presence.textile

```diff
@@ -0,0 +1,13 @@
+---
+title: Presence
+section: general
+index: 27
+---
+
+Ably's presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel) in real time using our "realtime SDKs":/realtime, or via the "Reactor service":https://www.ably.io/reactor. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the "REST API":/rest-api#presence.
```

Comment:
For the last line, "You can also request...", I'd say we should stick to the initial third person descriptive style. So for example "In addition, presence allows for a list of clients or devices that are online/offline on a channel at a particular point in time via the "REST API":/rest-api#presence to be obtained."

Reply:
I'd disagree, Tom, since we are talking about a product/feature, as a narrator. wdyt?

Reply:
Sure then, when I read it I read it differently, as a description of the service rather than describing the service. From the describing perspective makes sense.